tests/api_tests/test_participant_summary_api.py

```diff
@@ -4452,49 +4457,78 @@ def test_updated_since_parameter(self):
             for unexpected_participant in unexpected_participant_list
         ))
 
-    def test_remote_id_verified(self):
-        """ Test to see if a remote ID verification True Response saves successfully """
+    def test_remote_id_verification(self):
+        """ Test to see if a remote ID verification True, False, and No Response saves successfully """
+        # Set up participant, questionnaire, questionnaire response & send POST request to API
         participant = self.data_generator.create_database_participant()
         participant_id = f'P{participant.participantId}'
         authored = datetime.datetime.now()
         self.send_consent(participant.participantId, authored=authored)
+        # load & send a response w/ no remote id verification
+        questionnaire_id = self.create_questionnaire("questionnaire1.json")
+        resource = load_qr_response_json(
+            'questionnaire_response1.json',
+            questionnaire_id,
+            participant_id
+        )
+        self.send_post("Participant/%s/QuestionnaireResponse" % participant_id, resource)
+        # Get request from API to assert certain values stay UNSET
+        summary = self.send_get("Participant/%s/Summary" % participant_id)
+        self.assertEqual(summary['remoteIdVerificationStatus'], UNSET)
+        self.assertEqual(summary['onsiteIdVerificationType'], UNSET)
+        self.assertEqual(summary['onsiteIdVerificationVisitType'], UNSET)
+        self.assertEqual(summary['remoteIdVerificationOrigin'], UNSET)
+        self.assertEqual(summary['remoteIdVerifiedOn'], UNSET)
+        self.assertEqual(summary['everIdVerified'], UNSET)
+        self.assertEqual(summary['firstIdVerifiedOn'], UNSET)
+        self.assertEqual(summary['idVerificationOrigin'], UNSET)
+        # load & send a True remote ID verification response
         questionnaire_id = self.create_questionnaire("remote_id_verification_questionnaire.json")
         resource = load_qr_response_json(
             'remote_id_verification_questionnaire_response.json',
             questionnaire_id,
             participant_id
         )
         self.send_post("Participant/%s/QuestionnaireResponse" % participant_id, resource)
-        # Get request from API to assert information is accurate
+        # Get request from API to assert True Response information is accurate
         summary = self.send_get("Participant/%s/Summary" % participant_id)
         self.assertEqual(summary['remoteIdVerificationOrigin'], 'example')
-        self.assertEqual(summary['remoteIdVerificationStatus'], True)
+        self.assertEqual(summary['remoteIdVerificationStatus'], 'True')
```

Comment:
Not clear why this (and line 4503, 4516...) is being tested as the string 'True', as opposed to `assertTrue` ?  These are Boolean columns in the model?

Reply:
Yes, these are booleans in the db, but returning as strings in the API to be consistent. Recommended by @kenny-skaggs [here](https://pmi-engteam.slack.com/archives/C03TQ7D42RL/p1688764363292639).